include/sys/util.h

```diff
@@ -845,6 +883,55 @@ u8_t u8_to_dec(char *buf, u8_t buflen, u8_t value);
 #define MACRO_MAP_13(macro, a, ...) macro(a)MACRO_MAP_12(macro, __VA_ARGS__,)
 #define MACRO_MAP_14(macro, a, ...) macro(a)MACRO_MAP_13(macro, __VA_ARGS__,)
 #define MACRO_MAP_15(macro, a, ...) macro(a)MACRO_MAP_14(macro, __VA_ARGS__,)
+#define MACRO_MAP_16(macro, a, ...) macro(a)MACRO_MAP_15(macro, __VA_ARGS__,)
```

Comment:
Updating MACRO_MAP_CAT() to support the same number of arguments will probably avoid trouble

Reply:
ok